showcase/gapic-showcase/pom.xml

```diff
@@ -229,5 +229,10 @@
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-iam-v1</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Added since there are compilation issues for MockIAMPolicy and MockIAMPolicyImpl